dc

```diff
@@ -10,6 +10,24 @@ if [ -f .env ]; then
   IFS=$'\n'
   for x in $(grep -v '^#.*' .env); do export $x; done
   IFS=$OLD_IFS
+else
+  # ensure a .env is present
+  touch .env
+fi
+
+if [ -z $(cat .env | grep -i '^SERVER_UID=') ];
```

Comment:
this takes care of a `.env` file which does not mention a `SERVER_UID`

Reply:
This could be `$(grep -i '^SERVER_UID=' .env)` I think. `grep` already supports specifying a file.